src/components/Asset/AssetActions/Download.tsx

```diff
@@ -206,6 +206,10 @@ export default function Download({
       <div className={styles.info}>
         <div className={styles.filewrapper}>
           <FileIcon file={file} isLoading={fileIsLoading} />
+          <div className={styles.totalOrders}>
+            Total Orders:{' '}
+            <span className={styles.totalAmount}>{asset.stats.orders}</span>
```

Comment:
Think we should use asset.stats.consume here, in order to get rid of the build errors